### PR TITLE
Loki/Logs: Make it possible to copy log values to clipboard

### LINKED ISSUE #50904: Adding copy icon to label values
### Background / Why we're doing this

**Problem**
Today the user has to click, highlight and use a keyboard shortcut to copy a label value.

### Deliverables

**Solution**
We'd like to add a copy icon next to the log label value (and detected field value) to enable the user to copy the value by only applying one click.

The "copy" icon will only be visible when the user hovers over the log label value (in the example below, the value is `hm-east-2`). 

Here's a mockup of what it would look like after this change:
**Hover state**
Bildschirmfoto 2022-06-15 um 19 20 42
**Spacing**
Bildschirmfoto 2022-06-15 um 19 11 37

### PR DESCRIPTION
**Pull request by Seyaji**
Screenshot 2022-06-16 at 05 16 28

Fixes #50904

As requested in the above issue I have made the copy icon appear while hovering over a log value. 
When the icon is clicked it copies the value to the clipboard.

Hope this is ok :)

### CODE REVIEW

**Review by leventebalogh — COMMENTED** [review]
Thanks for your contribution @Seyaji! 🎉  

Left 2 comments, let me know if you have any questions.

File: packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx [hunk 1]
```diff
@@ -166,8 +177,23 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
...
           {parsedValue}
+          {mouseOver && (
+            <span className={styles.hoverValueCopy}>
+              <IconButton
+                name="copy"
+                title="Copy this value to clipboard"
+                onClick={() => {
+                  navigator.clipboard.writeText(parsedValue);
+                }}
+              />
```
[the remark this answers is not in the document]
Comment (Seyaji) [reply packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx:194]: Sounds good to me, sorry i didn't know about that button 😅

**Comment by Seyaji** [comment]
Got round to those changes!

Quick video of it in action:

https://user-images.githubusercontent.com/67058118/174888755-519a6782-909e-44bc-9364-00b91e7d72d2.mov

here's a screenshot of light mode:
Screenshot 2022-06-21 at 21 04 00


There is a slight difference between the filter icons on the left and the copy icon on the right. I'm not sure how to override the button highlight and / or animation to match them.
https://user-images.githubusercontent.com/67058118/174888188-d97ee41b-9712-4cf7-a678-63a1d00ccb3a.mov

**Review by leventebalogh — APPROVED** [review]
Nice one @Seyaji! 👏 

Sorry for not getting back for some time, got a bit snowed under.

Tested it out as well, works like a charm. Maybe one improvement that we could do in the future is to show a disappearing label once you clicked the copy-to-clipboard button, something like "Coped to clipboard" (as a confirmation towards the user that the action actually happened). That can have it's own PR I think though.

(I have pushed one commit where I have removed the commented out code)

**Comment by leventebalogh** [comment]
@Seyaji, have you signed the CLA (Contributor License Agreement)?

**Comment by leventebalogh** [comment]
(Closed/reopened the PR to trigger the CLA check as it got stuck)

**Comment by Seyaji** [comment]
@leventebalogh Thanks for reviewing this! No worries at all, I'm sure your'e quite busy. I think a disappearing comment box is a great Idea to give some visual conformation to the user.

**Pull request merged**